packages/angular/src/schematics/library/library.spec.ts

```diff
@@ -1187,51 +1187,58 @@ describe('lib', () => {
         );
 
         const eslintConfig = readJsonInTree(tree, 'libs/my-lib/.eslintrc.json');
-
-        expect(eslintConfig.overrides).toMatchInlineSnapshot(`
-          Array [
-            Object {
-              "extends": Array [
-                "plugin:@nrwl/nx/angular",
-                "plugin:@angular-eslint/template/process-inline-templates",
-              ],
-              "files": Array [
-                "*.ts",
-              ],
-              "parserOptions": Object {
-                "project": Array [
-                  "libs/my-lib/tsconfig.*?.json",
+        expect(eslintConfig).toMatchInlineSnapshot(`
```

Comment:
This diff is purely the change to make the snapshot to capture the whole config, instead of just `overrides`